### PR TITLE
fix(chat): fix tokenizer issue (Issue #2753)

### PR DESCRIPTION
**Pull request by Gimir**
**Description:**

- add js-tiktoken package (deduplication made)
- replace getEncoding method which was using wasm with browser supported method from js-tiktoken

Issues:

- Issue #2753 

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by Gimir** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2763.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat/src/hooks/useTokenizer.ts [hunk 1]
```diff
@@ -2,23 +2,17 @@ import { useCallback, useEffect, useState } from 'react';
 
 import { DialAIEntityModel } from '../types/models';
 
-import { Tiktoken, get_encoding } from '@dqbd/tiktoken';
+import { Tiktoken, getEncoding } from 'js-tiktoken';
```
Comment (Alexander-Kezik) [inline apps/chat/src/hooks/useTokenizer.ts]: @dqbd/tiktoken
we still have places which use this lib. Do we need them both: `js-tiktoken` and `@dqbd/tiktoken`. Can we get rid of `@dqbd/tiktoken`?

**Review by IlyaBondar — DISMISSED** [review]

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2763.nightly-test.deltixhub.io
>E2E tests status: failed

**Comment by Gimir** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2763.nightly-test.deltixhub.io
>E2E tests status: success

**Review by IlyaBondar — APPROVED** [review]

**Review by irinakartun — APPROVED** [review]

**Pull request merged**